src/release/breaking-changes/insert-content-text-input-client.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Insert Content Text Input Client
+description: Add a new method to the TextInputClient interface to allow Android virtual keyboards to
+insert rich content into Flutter TextFields. 
+---
+
+## Summary
+
+Adds `insertContent` method to the `TextInputClient` interface to allow Android's image keyboard 
+feature to insert content into a Flutter `TextField`.
+
+
+## Context
+
+As of Android 7.1, IMEs (input method editors or virtual keyboards) can send
+images and rich content into a text editor. This allows users to insert gifs,
+stickers, or context-aware rich content into a text field.
+
+
+## Description of change
+
+When the user inserts rich content via the IME, the platform sends a `commitContent` channel message
+called `TextInputClient.commitContent`.
+This notifies the Dart code that the IME inserted rich content.
+The channel message contains the mime type, URI, and bytedata for the inserted content in JSON form.
+
+
+## Migration guide
+
+If you had implemented `TextInputClient`, you must override
+`insertContent` to either support rich content insertion or provide an empty implementation.
```

Comment:
If they provide an empty implementation, what happens? The rich content is
still added, yes?

```suggestion
If you previously implemented the `TextInputClient` interface, you must override
`insertContent` to either support rich content insertion or provide an empty implementation.
```